packages/css/src/components/avatar/avatar.scss

```diff
@@ -0,0 +1,74 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+.ams-avatar {
+  aspect-ratio: 1/1;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  border-radius: 50%;
+  display: inline-flex;
+  font-family: var(--ams-avatar-font-family);
+  font-size: var(--ams-avatar-font-size);
+  justify-content: center;
+  line-height: var(--ams-avatar-line-height);
+  padding-block: var(--ams-avatar-padding);
+  padding-inline: var(--ams-avatar-padding);
+  width: calc(var(--ams-avatar-line-height) * var(--ams-avatar-font-size));
+}
+
+.ams-avatar--blue {
+  background-color: var(--ams-avatar-blue-background-color);
+  background-image: var(--ams-avatar-blue-background-image);
```

Comment:
If we render the default Icon as an SVG in the React component, we can set the color with CSS I think. This would get rid of the need for the encoded SVGs in the tokens.

Reply:
Background Images have been removed for Icon and Image components.